src/main/webapp/index.html

```diff
@@ -1,10 +1,18 @@
 <!DOCTYPE html>
 <meta charset="UTF-8">
+<div id="navbar-container"></div>
 <title>Open Sesame</title>
 <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 <link rel="stylesheet" href="style.css">
 <script src="script.js"></script>
 
```

Comment:
```suggestion
<title>Open Sesame</title>
<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
<link rel="stylesheet" href="style.css">
<script src="script.js"></script>

<div id="navbar-container"></div>
```

I'm not 100% sure because I haven't worked with HTML that doesn't have the additional tags like `<body>` and `<head>`, but I feel that the div for the navbar should most likely come after the meta information like the title of the page and the stylesheets.